doom/src/main/java/net/nullsum/freedoom/EntryActivity.java

```diff
@@ -54,18 +50,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
         AppSettings.reloadSettings(getApplication());
 
-        res = this.getResources();
+        Resources res = this.getResources();
         GamePadFragment.gamepadActions = Utils.getGameGamepadConfig(res);
 
-        actionBar.addTab(actionBar.newTab().setText(R.string.app_name).setTabListener(new TabListener<LaunchFragmentGZdoom>(this, "Gzdoom", LaunchFragmentGZdoom.class)));
```

Comment:
Eh wat? why remove types here? I guess it should still work, but oh well.

Reply:
The Android Studio IDE said it was now redundant, so I removed it. The IDE can be wrong sometimes, but it doesn't happen often. All of the Java 1.8 changes I made were the IDE's suggestions... so yeah